src/utils.jl

```diff
@@ -288,3 +261,58 @@ function range_shuffle!(r::UnitRange, a::AbstractVector; seed::Int=-1)
     end
 end
 
+"""
+    randbn(n, p, seed=-1)
+
+Return a binomally-distribted random number with parameters `n` and `p` and optional `seed`.
+
+### References
+- "Non-Uniform Random Variate Generation," Luc Devroye, p. 522. Retrieved via http://www.eirene.de/Devroye.pdf.
+- http://stackoverflow.com/questions/23561551/a-efficient-binomial-random-number-generator-code-in-java
+"""
+function randbn(n::Integer, p::Real, rng::AbstractRNG)
+    log_q = log(1.0 - p)
+    x = 0
+    sum = 0.0
+    while true
+        sum += log(rand(rng)) / (n - x)
+        sum < log_q && break
+        x += 1
+    end
+    return x
+end
+
+"""
+    is_graphical(degs)
+
```

Comment:
similarly, this doesn't use graphs so it belongs here.

Reply:
Do we still export this function? Because, while it does not use any graphs, it takes a list of degrees and determines if a graph with such a vertex sequence can even exist.

Reply:
we don't right now, but it's possible to do s.

Reply:
I exported it and added some tests.